cypress/integration/mob-expand-spec.js

```diff
@@ -0,0 +1,38 @@
+import { Preview } from '../page-objects/preview-page'
+
+const preview = new Preview()
+
+describe( 'Check the Expanded Layout in Mobile View', () => {
+
+	beforeEach( 'Open the English Page', () => {
+		cy.navigateToMobileTestPage()
+		cy.viewport( 550, 750 )
```

Comment:
do you plan to moving this line into the command `navigateToMobileTestPage`? so we don't need to repeat the viewport command in every spec.

Reply:
Refactor the code and added the viewport in the command navigateToMobileTestPage